data/mods/petmod/rulesets.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/**@type {{[k: string]: ModdedFormatsData}} */
+let BattleFormats = {
+	standardpetmod: {
+		inherit: true,
+		ruleset: ['[Gen 7] OU', '!Species Clause'],
+		banlist: ['All Pokemon'],
+		onBegin() {
+			// The only validator rule this currently modifies is Species Clause, so the added rule is just this
+			this.add('rule', 'PM Species Clause: Limit one of each Pokémon');
```

Comment:
Why not Clean Slate? Will all pet mods have the same species clause?

Reply:
No, all pet mods do not have the same species clause, it's just this one that needs the modified one.

Reply:
Then it should not called "PM". Also the mod should not be called "petmod".

Reply:
I’m calling it petmod for ease of client access for future pet mods

Reply:
anyways, fixed